.github/workflows/submodule-pin-check.yml

```diff
@@ -25,7 +25,8 @@ jobs:
         run: |
           status_state="pending"
           declare -Ar exceptions=(
-            [contracts]=origin/v2-main
+            [contracts]=origin/develop
```

Comment:
I assume we'd like to move this to `origin/main` once the blockchain eng team is good with it. @godzillaba, @gzeoneth, and @yahgwai FYI